### PR TITLE
FS: Moshe-edit_chef_menu_items-#46

### PR DESCRIPTION
**Pull request by mssiegel**
Ticket #46 

Description:

- Chefs can create meals with images
- Chefs can edit their meal items from their profile page
- Chefs can add multiple meal items on their profile page
- All edited images are "cache busted" so they change immediately upon being edited

Testing:

- Create new user
- Click button and fill out meal form to become a chef
- Go to your profile page
- Add a meal item without an image and verify placeholder image shows
- Add new meal items with images
- Update the meal items

### CODE REVIEW

**Comment by mssiegel** [comment]
Just made a new commit that removed an unneccessary function for getting images from request body on backend.

**Comment by mssiegel** [comment]
Just made a commit to removed unused "description" field from meal items

File: client/src/components/ChefSearch.js [hunk 1]
```diff
@@ -70,8 +70,8 @@ const ChefSearch = ({ coords }) => {
             Available Chefs:
           </Typography>
           <Box className={classes.chefTileArea}>
-            {chefs.map((chef) => (
-              <CustomCard chef={chef} />
+            {chefs.map((chef, i) => (
+              <CustomCard chef={chef} key={i} />
```
Comment (skassam21) [inline client/src/components/ChefSearch.js:74]: Good job fixing all of these warnings 👍

File: client/src/components/CreateMealForm.js [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+    becomeChef,
+    state: { user },
+  } = useContext(AuthContext);
+
+  const [cuisines, setCuisines] = useState([]);
+  const [selectedCuisine, setSelectedCuisine] = useState(null);
+
+  useEffect(() => {
+    (async function getCuisines() {
+      const { data: allCuisines } = await API.get("/cuisines");
```
Comment (skassam21) [inline client/src/components/CreateMealForm.js]: Maybe we should be putting all the cuisines in a context so we don't need to keep re-fetching it everywhere

File: client/src/components/CreateMealForm.js [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+              multiline = false,
+              rows,
+            },
+            index
+          ) => (
+            <FormControl key={index}>
+              <TextField
+                variant="outlined"
+                margin="normal"
+                required
```
Comment (skassam21) [inline client/src/components/CreateMealForm.js]: I noticed when you demo, that all the fields are shown as required, when they are not required. We should make that match (and if you remove noValidate from the form, we can actual just use HTML forms to validate)

**Pull request merged**